### ISSUE TITLE
Surrogate models used in black-box attack evaluation

### ISSUE THREAD

**Issue by yj373** [AI-secure/Transferability-Reduced-Smooth-Ensemble#8]
Hello, 

Thank you for sharing the code! I have some questions about the black-box evaluation:

1. Are the surrogate models simply trained with the "Naive_Trainer" in the trainer.py?
2. Could you please also share the parameters (learning rate, optimizer, number of epochs) you used for the training of the surrogate models?

By the way, the demo of DVERGE still does not work. I believe it is because of the inconsistency between the cifar_resnet.py and the resnet.py of the original repository of DVERGE. Thanks!
